docs/build_docs.py

```diff
@@ -10,6 +10,7 @@
 parser.add_argument('--prefix', default='build')
 parser.add_argument('--work_dir', default='.doctrees')
 parser.add_argument('--builder', default='html')
+parser.add_argument('--no-setup', action='store_true', default=False)
```

Comment:
Maybe we could make our parsers more robust for these kinds of updates if we were to use `parse_known_args` rather than `parse_args`. As is we would need to change `build_docs.py` in all our repos before we could merge this.

Reply:
great! I didn't know about `parse_known_args`, thanks!
i'll change all the other PRs.